schema/assets/video-frame.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Video Frame",
+    "description": "Asset representing a frame from a video at a specific timestamp.",
+    "allOf": [
+        {
+            "$ref": "#/$defs/assets/asset"
+        },
+        {
+            "$ref": "#/$defs/helpers/slottable-object"
+        },
+        {
+            "type": "object",
+            "properties": {
+                "ty": {
+                    "title": "Type",
+                    "description": "Asset type",
+                    "type": "integer",
+                    "const": 3
+                },
+                "vsid": {
+                    "title": "Video Asset ID",
+                    "description": "ID of the video asset to extract frame from",
+                    "type": "string"
+                },
+                "t": {
+                    "title": "Timestamp",
+                    "description": "Timestamp in seconds where to extract the frame from the video",

```

Comment:
How does this interact with start-time/time-stretch/general time remapping in referencing layers?

Also actual time units seem a bit odd since everything else in Lottie is expressed as frame indices (ip, op, etc).

Reply:
Actually, you are right, we can define time in frames here